src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php

```diff
@@ -636,6 +636,23 @@ public function fillSecretField($field, $value)
         $this->fillField($field, $decryptedValue);
     }
 
+    /**
+     * Function used to create data that contains sensitive credentials in a <createData> <field> override.
+     * The data is decrypted immediately prior to data creation to avoid exposure in console or log.
+     *
+     * @param string $command
+     * @throws TestFrameworkException
+     * @return void
+     */
+    public function magentoCliSecret($command)
```

Comment:
Add in parameter `$arguments = null` to match the signature of `magentoCLI`.
Also make sure this function name matches the casing on the other function `magentoCLI`

Reply:
Updated.